crates/oxc_cli/src/options.rs

```diff
@@ -0,0 +1,3 @@
+pub struct CliOptions {
```

Comment:
in hindsight maybe its a bit overkill to put this in it's own file, although more CliOptions may be implemented over time, so I'm happy to hear your thoughts on this

Reply:
This is good. We should be able to implement the `From` trait for clap commands.
So the `main` function will become smaller again.